packages/zen-bull-netting/src/ZenBullNetting.sol

```diff
@@ -695,11 +648,7 @@ contract ZenBullNetting is Ownable, EIP712, FlashSwap {
 
                     wethAmountToSend =
                         depositReceipt.amount * memVar.remainingEth / _params.depositsToProcess;
-                    if (wethAmountToSend > 1e12) {
-                        payable(depositReceipt.sender).sendValue(wethAmountToSend);
-                    } else {
-                        wethAmountToSend = 0;
-                    }
+                    payable(depositReceipt.sender).sendValue(wethAmountToSend);
```

Comment:
did we have to remove this due to bytecode size?

Reply:
yeah, but can be added now as there is enough space